### DISCUSSION TITLE
Cannot Connect To Repository

### DISCUSSION THREAD

**Discussion by saber-oz** [croneter/PlexKodiConnect#1371] [Q&A] [answered]
I currently have PKC installed and has been working fine but then kodi auto updated to Matrix the other day and PKC stopped working.
It seems that it was not updating for some reason.

So I tried to add in the repos again but keep getting the pop up Could Not Connect to repository 
I have tried both installing the repos directly from the Zip file as well as adding the the source to File manager.
I have tried installing all 3 repos to test and I get the same result

It installs the repo above no worries, it is just when I click on them to open it doesn't work.

Any thoughts as to what might be causing this.

2021-02-26 07:18:11.331 T:18399   ERROR <general>: Open - Unhandled exception
2021-02-26 07:18:11.331 T:18399   ERROR <general>: Open - Error opening https://raw.githubusercontent.com/croneter/binary_repo/master/stable/addons.xml.md5
2021-02-26 07:18:11.331 T:18399   ERROR <general>: CRepository: failed read 'https://raw.githubusercontent.com/croneter/binary_repo/master/stable/addons.xml.md5'
2021-02-26 07:18:12.539 T:18362 WARNING <general>: CGUIMediaWindow::OnMessage - updating in progress
2021-02-26 07:18:13.138 T:18744   ERROR <general>: GetDirectory - Error getting addons://repository.plexkodiconnect/
2021-02-26 07:18:13.153 T:18362   ERROR <general>: CGUIMediaWindow::GetDirectory(addons://repository.plexkodiconnect/) failed

UPDATE...
Just a quick update. I tried to manually install PKC and when I tried to install kodi adons I get this error.

CAddonInstallJob[plugin.video.plexkodiconnect.movies]: The dependency on xbmc.python version 2.1.0 could not be satisfied.

I assume it has to do with Kodi using V3 of python now

**Comment by croneter** [comment, marked as answer]
You need to manually install a new pkc repository. see here https://github.com/croneter/PlexKodiConnect/issues/1359

**Reply by croneter** [reply]
Is this resolved, @saber-oz ?

**Reply by saber-oz** [reply]
Hey @croneter 

I actually did try manually installing the repo but it was still not working.... however

After quite some digging and googling it turned out to be something else.
A while back I had increased the video memory cache to try and help with buffering 4K videos.
However must have set it to high.
This was causing the unhandled exception in the error log.
Once I reduced this memory buffer down your repo now opens.

It only seems to effect external ropo's as the internal Kodi one kept working.

Anyway long story short it is now installed and back up and running.
Thx
